server/ladder_service/ladder_service.py

```diff
@@ -221,6 +225,48 @@ async def fetch_matchmaker_queues(self, conn):
                 errored.add(name)
         return matchmaker_queues
 
+    async def fetch_rating_peak(self, conn, rating_type):
+        result = await conn.execute(
```

Comment:
Why pass the conn as a parameter? Seems like this method should create the connection and use it.

Reply:
This is a remnant of some refactoring. I'll fix it